### PR TITLE
Add 2022 data entry

### PR DESCRIPTION
**Pull request by grc54**
including codebook, code and output data

### CODE REVIEW

File: Code/2022/data_2022.Rmd [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+```{r packages}
+library(here)
+library(tidyverse)
+library(haven)
+```
+
+Use the Haven package to read in the SPSS files.
+
+```{r read in spss sav files}
+urls <- read_csv(here("Code/download_urls.csv")) |>
```
Comment (kevinwang09) [inline Code/2022/data_2022.Rmd:21]: This file was not updated as part of this PR.

File: README.md
Comment (kevinwang09) [inline README.md:1]: Revert the changes in this README. I updated the repo at the same time as you did, so the changes should be pulled in. The main changes were mostly the use of past tense. And also making it clear that we are using this repo for data cleaning and the actual package is available elsewhere.

**Review by kevinwang09 — CHANGES_REQUESTED** [review]
I also want to see changes to the `Code/student_bind_rows.Rmd`, `Code/countrycode_bind_rows.Rmd` and `Code/school_bind_rows.Rmd` files, where data across different years are further processed and compressed into a single `.rda` file.

File: Code/2022/data_2022.Rmd (hunk 1 above)
Comment (grc54) [reply Code/2022/data_2022.Rmd:21]: download_urls file is now uploaded, we'll work on the other files afterward

File: README.md
Comment (grc54) [reply README.md:1]: the README is now updated

**Review by kevinwang09 — APPROVED** [review]
Approve the code, however, the xlsx table of variables needs some subtaintial changes. Opening new issues.
